index.bs

```diff
@@ -1132,12 +1132,21 @@ Methods</h4>
 			Note: Multiple <em>decoding threads</em> can run in parallel to
 			service multiple calls to <code>decodeAudioData</code>.
 
-			1. Attempt to decode the encoded {{BaseAudioContext/decodeAudioData(audioData, successCallback, errorCallback)/audioData!!argument}} into linear
-				PCM.
+			1. Let <var>canDecode</var> be a boolean flag, initially set to true.
```

Comment:
Let's use `<var>can decode</var>`. Let us not camel-casing the algorithm variables from now on.

Reply:
done.